microsoft-365/enterprise/ms-cloud-germany-transition-add-experience.md

```diff
@@ -83,73 +58,34 @@ Between Phase 2 and phase 3, Partner Portal may not be accessible. During this t
     Question from ckinder
     The following paragraph is not clear
 -->
+## Custom DNS Updates
 **Applies to:**  All customer managing their own DNS zones
 
 | Step(s) | Description | Impact |
 |:------|:-------|:-------|
 | Update on-premises DNS services for Office 365 services endpoints. | Customer-managed DNS entries that point to Microsoft Cloud Deutschland need to be updated to point to the Office 365 Global services endpoints. | Failure to do so may result in failure of the service or of software clients. |
 ||||
 
-**Applies to:** Customers using  third-party services for Office 365 services endpoints
+## 3rd Party Services
+**Applies to:** Customers using third-party services for Office 365 services endpoints
 
 | Step(s) | Description | Impact |
 |:-------|:-------|:-------|
 | Update partners and third-party services for Office 365 services endpoints. | <ul><li>Third-party services and partners that point to Office 365 Germany need to be updated to point to the Office 365 services endpoints. Example: Re-register, in alignment with your vendors and partners, the gallery app version of applications, if available. </li><li>Point all custom applications that leverage Graph API from `graph.microsoft.de` to `graph.microsoft.com`. Other APIs with changed endpoints also need to be updated, if leveraged. </li><li>Change all non-first-party enterprise applications to redirect to the worldwide endpoints. </li></ul>| Required action. Failure to do so may result in failure of the service or of software clients. |
 ||||
 
-### SharePoint Online post migration
+## SharePoint Online
+**Applies to**: Customers using SharePoint 2013 Workflows
 
 | Step(s) | Description | Impact |
 |:-------|:-------|:-------|
 | Republish SharePoint 2013 workflows. | In the pre-migration work, we reduced the number of SharePoint 2013 workflows. Now with migration complete, the customer can republish the workflows. | This is a required action. Failure to do so may result in user confusion and help desk calls. |
 | Share items via Outlook | Sharing items in  SharePoint Online and OneDrive for Business via Outlook no longer works after tenant cutover. |<ul><li>In SharePoint Online and OneDrive for Business, you can share items via Outlook. After pressing the Outlook button, a shareable link is created and pushed into a new message in the Outlook Web App.</li><li>After tenant cutover, this method of sharing won't work. We recognize this is a known issue. However, since this Outlook feature is in the path of deprecation, fixing the issue is not planned until the deprecation is rolled out. </li></ul>|
```

Comment:
```suggestion
| Share items via Outlook | Sharing items in SharePoint Online and OneDrive for Business via Outlook no longer works after tenant cutover. |<ul><li>In SharePoint Online and OneDrive for Business, you can share items via Outlook. After pressing the Outlook button, a shareable link is created and pushed into a new message in the Outlook Web App.</li><li>After tenant cutover, this method of sharing won't work. We recognize this is a known issue. However, since this Outlook feature is in the path of deprecation, fixing the issue is not planned until the deprecation is rolled out. </li></ul>|
```